src/ui-test/tests/remote.debug.on.openshift.test.ts

```diff
@@ -0,0 +1,245 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License", destination); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as path from 'path';
+import { ActivityBar, BottomBarPanel, Breakpoint, DebugView, DefaultWait, EditorView, InputBox, SideBarView, TextEditor, VSBrowser, WebDriver } from "vscode-uitests-tooling";
+import { assert } from 'chai';
+import { addNewItemToRawJson, clearTerminal, deleteFolderContents, disconnectDebugger, executeCommand, killTerminal, waitUntilEditorIsOpened, waitUntilTerminalHasText } from '../utils';
+import { DEMO_FILE, QUARKUS_ATTACH_DEBUGGER, QUARKUS_CREATE_COMMAND, QUARKUS_DIR, QUARKUS_PROJECT_FOLDER, SPRINGBOOT_ATTACH_DEBUGGER, SPRINGBOOT_CREATE_COMMAND, SPRINGBOOT_DIR, SPRINGBOOT_PROJECT_FOLDER } from '../variables';
+
+
+describe.only('Remote debug on OpenShift', function () {
+    this.timeout(200000);
+
+    let driver: WebDriver;
+    let textEditor: TextEditor;
+
+    describe('Spring Boot project', function () {
+
+        before(async function () {
+            await prepareEnvironment('springboot', SPRINGBOOT_DIR);
+        });
+
+        after(async function () {
+            await cleanUp(SPRINGBOOT_DIR);
+        });
+
+        it(`SpringBoot`, async function () {
+            driver = VSBrowser.instance.driver;
+
+            await createCamelRoute(driver, "Demo");
+            await createProject(driver, SPRINGBOOT_CREATE_COMMAND)
+
+            // open file
+            await VSBrowser.instance.openResources(path.join(SPRINGBOOT_PROJECT_FOLDER, DEMO_FILE));
+
+            // set breakpoint
+            textEditor = new TextEditor();
+            await driver.wait(async function () {
+                return await textEditor.toggleBreakpoint(15);
+            }, 5000);
+
+            // open debug view
+            const debugView = await openDebugView();
+
+            // validate configuration and start debug
+            assert.isTrue(await validateLaunchConfiguration(debugView, SPRINGBOOT_ATTACH_DEBUGGER));
```

Comment:
I think it would be convenient to have a message for the assertion explaining what is checked